pixeltable/catalog/table.py

```diff
@@ -276,53 +266,47 @@ def count(self) -> int:
         """Return the number of rows in this table."""
         return self._df().count()
 
-    @property
     def columns(self) -> list[str]:
         """Return the names of the columns in this table."""
         cols = self._tbl_version_path.columns()
         return [c.name for c in cols]
 
-    @property
     def _schema(self) -> dict[str, ts.ColumnType]:
```

Comment:
Per discussion last week, it'd be clearer to call this `_get_schema()` if it's no longer a property.

I wonder if we should just call it `get_schema()` and make it a public method. It's come up a few times now that users have wanted to refer to the schema.

Or have a separate `get_schema()` that turns the `ColumnType` references back into `type`s? (Although, there are still a few other places in the code where `ColumnType` references leak to the user, if I recall correctly)